src/06-loops.js

```diff
@@ -117,7 +161,18 @@ function targetCount(array, target) {}
  *  firstIndexFound([ "left", "right", "left" ], "up");
  *  //> -1
  */
-function firstIndexFound(array, target) {}
+ function firstIndexFound(array, target) {
+  for(let i = 0; i < array.length;i++){
+    if( target ===  array[i]){
+      return i;
+      
+    }else if( target !==  array[i]){
```

Comment:
we don't need this `else if` here. Remember that by default JavaScript will continue our iteration if our first condition did not return anything.